From f50a4f6e244cfc8e773300c03aaf4db391f3028a Mon Sep 17 00:00:00 2001
From: Andrew Cooper <andrew.cooper3@citrix.com>
Date: Tue, 17 Dec 2019 13:49:56 +0000
Subject: libxc/restore: Fix error message for unrecognised stream version

The Expected and Got values are rendered in the wrong order.

Signed-off-by: Andrew Cooper <andrew.cooper3@citrix.com>
Acked-by: Wei Liu <wl@xen.org>

diff --git a/tools/libxc/xc_sr_restore.c b/tools/libxc/xc_sr_restore.c
index ea7b0339ef..1ac404b97b 100644
--- a/tools/libxc/xc_sr_restore.c
+++ b/tools/libxc/xc_sr_restore.c
@@ -36,7 +36,7 @@ static int read_headers(struct xc_sr_context *ctx)
     else if ( ihdr.version != IHDR_VERSION )
     {
         ERROR("Invalid Version: Expected %d, Got %d",
-              ihdr.version, IHDR_VERSION);
+              IHDR_VERSION, ihdr.version);
         return -1;
     }
     else if ( ihdr.options & IHDR_OPT_BIG_ENDIAN )
